usd/include/sdf/usd/sdf_parser/World.hh

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (C) 2022 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#ifndef SDF_USD_WORLD_HH_
+#define SDF_USD_WORLD_HH_
```

Comment:
Do we want to use an include guard that is more specific to the SDF -> USD parser? I ask because I imagine that for the USD -> SDF parser, we could also have a `World.hh` file. Perhaps we can use the format `SDF_TO_USD_<fileName>_HH_` for the SDF -> USD parser, and use the format `SDF_FROM_USD_<fileName>_HH_` for the USD -> SDF parser. If this sounds good, we can make the following change:
```suggestion
#ifndef SDF_TO_USD_WORLD_HH_
#define SDF_TO_USD_WORLD_HH_
```

Reply:
The usual pattern used here and in other ignition library is to use the path of the file changing `/` to `_` and capitalizing everything, so that would be my recommendation. i.e `SDF_USD_SDF_PARSER_WORLD_HH_`.

Reply:
I made a follow-up include guard comment in the USD -> SDF converter: https://github.com/ignitionrobotics/sdformat/pull/827#discussion_r791013232

Reply:
Sounds good :+1: I made this update in https://github.com/ignitionrobotics/sdformat/pull/817/commits/01aeb5747114f4449db417dca19a3d2300e516a7